### PR TITLE
vim-patch:8.0.{1531,1544,1589,1591,1712,1745,1747},8.1.{819},8.2.{420,539,893,894,895,899,1114,1118,1169,1170,1171,1172,1173,1177,1179,1180,1181,1187,1188,1196,1198,1211,1214,1215,1222}

### PR DESCRIPTION
**Pull request by janlazo**
Most are N/A patches but there is no `version.c` PR to put them.

I wonder if Github PR titles have a character limit.

### CODE REVIEW

**Comment by janlazo** [comment]
@teto Can you review this or ask someone else to review it so that we have a new version.c PR?

File: src/nvim/misc1.c [hunk 1]
```diff
@@ -1029,6 +1029,15 @@ void fast_breakcheck(void)
   }
 }
 
+// Like line_breakcheck() but check 100 times less often.
+void veryfast_breakcheck(void)
+{
+  if (++breakcheck_count >= BREAKCHECK_SKIP * 100) {
```
Comment (janlazo) [inline src/nvim/misc1.c:1035]: Only difference between the 3 `_breakcheck` functions is the multiplier.

**Comment by teto** [comment]
I dont have a good connectivity at the moment so not sure when I can review it. Anyone feel free to pick it up. If you can add patches in another PR, that would prevent from having to review a too long one afterwards.

File: src/nvim/spellfile.c [hunk 2]
```diff
@@ -3998,6 +4008,7 @@ static int deref_wordnode(spellinfo_T *spin, wordnode_T *node)
     for (np = node; np != NULL; np = np->wn_sibling) {
       if (np->wn_child != NULL)
         cnt += deref_wordnode(spin, np->wn_child);
+
```
Comment (teto) [inline src/nvim/spellfile.c:4011]: newline

**Review by teto — APPROVED** [review]
apart from the useless newline, looks perfect.

**Comment by janlazo** [comment]
Removed the newline in patch 8.2.0894 and rebased.

**Pull request merged**